crates/spirv-std/macros/Cargo.toml

```diff
@@ -14,4 +14,4 @@ proc-macro = true
 spirv-std-types.workspace = true
 proc-macro2 = "1.0.24"
 quote = "1.0.8"
-syn = { version = "1.0.58", features = ["full", "visit-mut"] }
+syn = { version = "1.0.85", features = ["full", "visit-mut"] }
```

Comment:
Should we consider switching to [`syn 2.0.0`](https://github.com/dtolnay/syn/releases/tag/2.0.0)?
I guess even if we do, I still want to land the CI check separately.

Reply:
yeah we can definitely switch